webapp/certified/views.py

```diff
@@ -102,12 +101,12 @@ def _parse_query_params(all_releases, all_vendors):
             "servers",
             "iot",
             "socs",
+            "Ubuntu Core",
```

Comment:
and same here, aren't `iot` and `IoT` redundant?

Reply:
Same answer as previous comment. We could additionally just do lowercase and have only `iot` I guess, but we need to accomodate for both due to UX vs Certification API reuirements.

Reply:
ok, looks like we can't do that either, because it messes up with Server SoC, you can't match server soc with Server SoC, because if you make the initial letter uppercase it's not enough. It's mainly because of the UX requirement this needs to be passed this way.